credentials/build.gradle.kts

```diff
@@ -13,25 +11,15 @@ repositories {
 }
 
 dependencies {
-  testImplementation(kotlin("test"))
-
   api("com.danubetech:verifiable-credentials-java:1.5.0")
-  api("com.nimbusds:nimbus-jose-jwt:9.34")
-  api("decentralized-identity:did-common-java:1.9.0")
-  api("decentralized-identity:uni-resolver-core:0.13.0")
 
-  implementation("com.github.richardbergquist:java-multicodec:main-SNAPSHOT")
-  implementation("com.google.crypto.tink:tink:1.10.0")
+  implementation(project(":dids"))
+  implementation(project(":common"))
+  implementation(project(":crypto"))
+  implementation("com.fasterxml.jackson.module:jackson-module-kotlin:2.13.0")
   implementation("com.nfeld.jsonpathkt:jsonpathkt:2.0.1")
-}
+  implementation("com.nimbusds:nimbus-jose-jwt:9.34")
```

Comment:
Hmmmm I'm not sure why this is needed here. Can you elaborate?

Reply:
it's being used in [`sign`](https://github.com/TBD54566975/web5-kt/blob/fd11022b52342ff5728ec62e7cd7d0a890f938ed/credentials/src/main/kotlin/web5/sdk/credentials/VerifiableCredential.kt#L62-L100) and [`verify`](https://github.com/TBD54566975/web5-kt/blob/fd11022b52342ff5728ec62e7cd7d0a890f938ed/credentials/src/main/kotlin/web5/sdk/credentials/VerifiableCredential.kt#L192-L247)